sources/PTYTab.m

```diff
@@ -2460,10 +2460,12 @@ - (double)blurRadius {
         }
     }
     if (count > 0) {
-        if (@available(macOS 10.13, *)) {
+        if (@available(macOS 10.14, *)) {
+            return sum / count;
```

Comment:
This is intended to cascade from macOS versions highest -> lowest due to the nature of `@available`. If we have 10.14, then we are assumed to be on 10.14-10.16. Clip the value for 10.13, and allow normal range for 10.12 and under.